### PR TITLE
remove support for PULL_SECRET variable

### PR DESCRIPTION
**Pull request by dhellmann**
Previous changes moved to using a CI_TOKEN for access to the openshift
CI image registry and a file for the other parts of the pull
secret. The PULL_SECRET variable was retained to allow the CI job to
be updated. That has been done, so now we can remove support for
PULL_SECRET.

### CODE REVIEW

**Comment by zaneb** [comment]
Question: does the pull secret expire more often or less often than the token?

**Comment by dhellmann** [comment]
> Question: does the pull secret expire more often or less often than the token?

As far as I can tell, the token is static. The pull secret seems to expire in ~24 hrs.

File: common.sh [hunk 1]
```diff
@@ -273,17 +273,15 @@ export PULL_SECRET_FILE=${PULL_SECRET_FILE:-$WORKING_DIR/pull_secret.json}
...
 # variable errors in the next 2 checks.
-export PULL_SECRET=${PULL_SECRET:-}
 export PERSONAL_PULL_SECRET=${PERSONAL_PULL_SECRET:-$SCRIPTDIR/pull_secret.json}
 export CI_TOKEN=${CI_TOKEN:-}
 
-# avoid "-z $PULL_SECRET" to ensure the secret is not logged
-if [ ! -s ${PERSONAL_PULL_SECRET} -a ${#PULL_SECRET} = 0 ]; then
+if [ ! -s ${PERSONAL_PULL_SECRET} ]; then
   error "${PERSONAL_PULL_SECRET} is missing or empty"
   error "Get a valid pull secret (json string) from https://cloud.redhat.com/openshift/install/pull-secret"
```
Comment (hardys) [inline common.sh:281]: Currently if you run with an old config, you just hit this error, which doesn't mention PULL_SECRET is no longer supported, and it's not super-obvious if you've not been following these PRs - perhaps we could detect the case where PULL_SECRET is still set, and output a helpful message or link to the docs?

Comment (dhellmann) [reply common.sh:281]: Good point. Done.

**Comment by hardys** [comment]
lgtm but needs a rebase

File: common.sh [hunk 2]
```diff
@@ -273,21 +273,30 @@ export PULL_SECRET_FILE=${PULL_SECRET_FILE:-$WORKING_DIR/pull_secret.json}
 
 # Ensure a few variables are set, even if empty, to avoid undefined
 # variable errors in the next 2 checks.
+export PERSONAL_PULL_SECRET=${PERSONAL_PULL_SECRET:-$SCRIPTDIR/pull_secret.json}
```
Comment (zaneb) [inline common.sh:276]: This line appears twice now.

Comment (dhellmann) [reply common.sh]: Fixed

**Review by zaneb — COMMENTED** [review]
/lgtm

**Pull request merged**